yolo_inference.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from detection_utilities import detect
+from inventory_state_utils import update_inventory_state
+
+
+class YoloExecutor:
+
+    def __init__(self):
+        print('doin tings')
+        self.labels = None
+        self.detections = []
+
+    def get_labels(self, input_frame: np.ndarray) -> dict:
+
+        print(f'Length of detections: {len(self.detections)}')
+
+        self.detections = detect(input_frame=input_frame, operation='add')   #   Array of tuples (x1, y1, x2, y2, conf, cls_conf, cls_pred) per image
+
+    def post_process_stream(self) -> dict:
+
+        print(self.detections)
+        instance_inventory_dict = {}
+
+        for item in self.detections:
```

Comment:
rename `item` to something more descriptive

Reply:
renamed to faces